homeassistant/components/weather/yweather.py

```diff
@@ -82,9 +88,17 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                       len(yahoo_api.yahoo.Forecast))
         return False
 
+    for cond, condlst in CONDITION_CLASSES.items():
+        for condi in condlst:
+            CONDITION_CLASSES_LIST[condi] = cond
```

Comment:
CONDITION_CLASSES_LIST should be a const, so I think I forget to move the init code into globle scope.